features/other_b/skin.feature

```diff
@@ -24,6 +24,21 @@ Feature: Non-public site and work skins
   When I go to the new skin page
     Then I should see "Sorry, you don't have permission"
 
+  Scenario: An admin can reject and unreject a skin
```

Comment:
This looks like a duplicate of this test from the admin skins feature:
https://github.com/otwcode/otwarchive/blob/ab4e40182099780635b7af479c249c68f3d2f721/features/admins/admin_skins.feature#L35-L48
I'm guessing the process of resolving merge conflicts went a bit awry.